src/client/main.cpp

```diff
@@ -48,7 +48,7 @@ std::string get_server_address()
 
 int main(int argc, char* argv[])
 {
-    QCoreApplication app(argc, argv);
+    QApplication app(argc, argv);
```

Comment:
Hmm, can you check if the command line runs when no display server is running (i.e. "DISPLAY=")

QApplication subclasses QGuiApplication, which loads Qt platform backends for various graphical shells. Those are needed for the systray icon ofc, but for a pure command-line util, I'm unsure if it'll run or just bail.

Reply:
No, it doesn't:(  Offhand, do you if there is some way to load the platfrom backends after the fact, like when only the systray command runs?  I'll do some research, but hoping you have a quick answer:)

Reply:
Backends load on Q{Gui,}Application construction.

One workaround might be to read the CLI very early looking for systray:
```
if (!argv_has_systray())
    setenv("QT_QPA_PLATFORM", "minimal");

QApplication app(argc, argv);
```

Reply:
That did the trick.  Thanks!

Reply:
Wouldn't it be better to have a separate command altogether then? Just a separate main instead of the if?

Reply:
@ricab,
I want to use the same binary, ie, `multipass` for the cli, systray icon, and the down-the-road full blown gui, so I think the simple `if` will do the trick.